swap_meet/electronics.py

```diff
@@ -1,2 +1,19 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+
+
+class Electronics(Item):
+
+    def __init__(self, id=None, type="Unknown", condition=0 ):
+        
+        super().__init__(id, condition)
+        
```

Comment:
When we only have a couple of lines in the function definition, adding space between each line can be a little much. Feel free to remove some of the whitespace here!